subprojects/instant-execution/src/main/kotlin/org/gradle/instantexecution/serialization/codecs/BuildOperationListenersCodec.kt

```diff
@@ -42,7 +42,8 @@ class BuildOperationListenersCodec {
 
         private
         val classNameWhitelist = setOf(
-            "org.gradle.trace.buildops.BuildOperationTrace${'$'}RecordingListener"
+            "org.gradle.trace.buildops.BuildOperationTrace${'$'}RecordingListener",
```

Comment:
:o: Consider adding a comment that this may be removed after some time given this listener doesn't exist anymore in `gradle-profiler`

Reply:
Yeah, I was wondering that. Should I remove the old listener right now?

Reply:
I added a comment.